bin/bootstrax

```diff
@@ -1302,21 +1341,17 @@ def cleanup_db():
 
     log.info("Checking for bad stuff in database")
 
-    # TODO
-    #  the lines below are obsolete given the changed strategy of how to use the bs_coll
-    # Bootstrax instances that say they are active, but haven't reported in for a while
-    # (these are on other hosts, or we would have killed them already)
-    # while True:
-    #     send_heartbeat()
-    #     bd = bs_coll.find_one_and_update(
-    #         {'host': hostname,
-    #          'status': {'$ne': dead_state},
-    #          'time': {'$lt': now(-timeouts['bootstrax_presumed_dead'])}
-    #          },
-    #         {'$set': {'state': dead_state}})
-    #     if bd is None:
-    #         break
-    #     log.info(f"Cleaned old boostrax entry on host {bd['host']}. Now presumed dead. Rest in peace")
+    # Check for all the ebs if their last state message is not longer ago than the time we assume that the eb is dead.
+    for eb_i in range(6):
+        bd = bs_coll.find_one(
```

Comment:
We should index this query, and also remove the old collections.

Reply:
If I'm not mistaken it is indexed in order to be TTL right?

Reply:
`time` is indexed (TTL), but `host` isn't (or wasn't, until I wrote this comment). Also, index intersection doesn't always work as desired or expected, but if we use separate indexes for `host` and `time` it should be fine.